tests/apm/bug0950-001.phpt

```diff
@@ -8,7 +8,7 @@ PHPC-950: Segfault killing cursor after subscriber HashTable is destroyed (no su
 <?php
 require_once __DIR__ . "/../utils/basic.inc";
 
-$manager = new MongoDB\Driver\Manager(STANDALONE);
+$manager = new MongoDB\Driver\Manager(URI);
```

Comment:
You *did* write a script for this one, right? :-)

Reply:
This was a scripted search and replace, but everything else was manual as it required adding c conditional skips (e.g. `REPLICASET` needed to require a replica set topology). After bulk-changing  these to `URI`, I then ran the suite against various topologies to determine which tests were not topology-agnostic.